modules/local/download_kofamscan_db.nf

```diff
@@ -1,5 +1,5 @@
 process DOWNLOAD_KODB {
```

Comment:
I think it's good to keep to the standard nf-core naming, and make sure that the name of processes reflect the name of the file's path.